tasks/main.yml

```diff
@@ -52,7 +51,7 @@
 - name: Ping outside
   command: "lxc-attach  -n {{ ct_started.msg[3:6] }} -- ping {{ ping_server }} -c 2"
   when: ct_created is changed
-  delegate_to: "{{ proxmox_node }}.quentindawans.com"
+  delegate_to: "{{ proxmox_node }}"
```

Comment:
I'm not sure this would work. My node don't use the fqdn. It might be something I can change in the `inventory`. I need to do a test.